src/pages/AddDog/AddDog.jsx

```diff
@@ -2,11 +1,426 @@
+import React, { useState } from 'react';
+import {
+  FormControl,
+  FormLabel,
+  Input,
+  Button,
+  ButtonGroup,
+  Heading,
+  Select,
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbLink,
+  Avatar,
+} from '@chakra-ui/react';
+import { ArrowBackIcon, ChevronRightIcon, AddIcon } from '@chakra-ui/icons';
+import './AddDog.css';
+import { useBackend } from '../../contexts/BackendContext';
 import Location from '../../components/Location';
 
 const AddDog = () => {
+  const [facilityid, setFacilityID] = useState(0);
+  const [groupnum, setGroupNum] = useState(0);
+  const [graddate, setGradDate] = useState('');
+  const [dogname, setDogName] = useState('');
+  const [age, setAge] = useState(0);
+  const [shelter, setShelter] = useState('');
+  const [breed, setBreed] = useState('');
+  const [chiptype, setChipType] = useState('');
+  const [chipnum, setChipNum] = useState(0);
+  const [gender, setGender] = useState('');
+  const [profilepic, setProfilePic] = useState('');
+  const [altname, setAltName] = useState('');
+  const [notes, setNotes] = useState('');
+  const [adoptername, setAdopterName] = useState('');
+  const [adopterphone, setAdopterPhone] = useState('');
+  const [addrline, setAddrLine] = useState('');
+  const [adoptcity, setAdoptCity] = useState('');
+  const [adoptstate, setAdoptState] = useState('');
+  const [zip, setZip] = useState('');
+  const [adoptemail, setAdoptEmail] = useState('');
+  const [fees, setFees] = useState(0);
+  const [revenue, setRevenue] = useState(0);
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'setRevenue' is assigned a value but never used.